docs/kubernetes/KubernetesIntegration.md

```diff
@@ -0,0 +1,38 @@
+# Integrating the Calico-Kubernetes Plugin
+
+This guide will take you through all of the configuration hooks needed to integrate the Calico-Kubernetes plugin into your cloud platform.
```

Comment:
How about "This guide will describe the configuration required to use the Calico network plugin in your Kubernetes deployment" since they aren't all hooks.